_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -11082,6 +11082,21 @@
 	icon_state = "darkgrey"
 	},
 /area/station/engineering/smes)
+"aUM" = (
+/obj/structure/cable/yellow{
+	d2 = 4;
+	icon_state = "0-4"
+	},
+/obj/machinery/power/apc{
+	dir = 8;
```

Comment:
APCs don't have directional sprites, so the dir varedit should be removed

Reply:
where will that be?

Reply:
all of the west bump APCs you added.

Reply:
fixed